tests/unit/ml/test_sql.py

```diff
@@ -184,11 +188,12 @@ def test_create_imported_model_produces_correct_sql(
     model_creation_sql_generator: ml_sql.ModelCreationSqlGenerator,
 ):
     sql = model_creation_sql_generator.create_imported_model(
+        dataset=bigquery.DatasetReference("a-project", "a-dataset"),
         options={"option_key1": "option_value1", "option_key2": 2},
     )
     assert (
         sql
-        == """CREATE TEMP MODEL `my_model_id`
+        == """CREATE MODEL `a-project`.`a-dataset`.`my_model_id`
```

Comment:
I'm intentionally choosing different projects here so that we know that we're creating in the dataset specified, not something hardcoded (not that would be likely, but still...)

Reply:
Done